client/app/main/main.html

```diff
@@ -11,35 +11,7 @@
 
 <div class="container">
   <div class="row text-area">
-    <div class="col-md-8">
```

Comment:
this isn't a indentation refactor. just moved the element. It's fine in this commit.